src/sentry/models/event.py

```diff
@@ -27,7 +27,7 @@ def __init__(self, data, skip_renormalization=False, **kwargs):
             # XXX: This is a hack to make generic events work (for now?). I'm not sure whether we
             # should include this in the rust normalizer, since we don't want people sending us
             # these via the sdk.
-            if pre_normalize_type == "generic":
+            if pre_normalize_type in ["generic", "nel"]:
```

Comment:
are there other places where we need to add this check where we're looking for `generic` now?

Reply:
Sorry, this is a temporary hack before we merge and release Relay part (https://github.com/getsentry/relay/pull/2421).
This will be removed after.